roles/srcds/files/addons/sourcemod/scripting/votescramble.sp

```diff
@@ -114,13 +123,13 @@ void AttemptVoteScramble(int client)
 
 	if (g_bVoted[client])
 	{
-		ReplyToCommand(client, "You have already voted for a team scramble. [%d/%d votes required]", g_iVotes, g_iVotesNeeded);
+		MC_ReplyToCommandEx(client, client, "[{green}SM{default}] {teamcolor}You {default}have already voted for a team scramble. [{lightgreen}%d{default}/{lightgreen}%d {default}votes required]", g_iVotes, g_iVotesNeeded);
 		return;
 	}
 
 	g_iVotes++;
 	g_bVoted[client] = true;
-	PrintToChatAll("%s wants to scramble teams. [%d/%d votes required]", name, g_iVotes, g_iVotesNeeded);
+	MC_PrintToChatAllEx(client, "[{green}SM{default}] {teamcolor}%s {default}wants to scramble teams. [{lightgreen}%d{default}/{lightgreen}%d {default}votes required]", name, g_iVotes, g_iVotesNeeded);
```

Comment:
@leighmacdonald at one point mentioned wanting to get rid of the `[SM]` prefix to chat messages, I likewise don't think it is necessary to have it.

Reply:
i think it can be useful to draw peoples' eyes towards things happening in chat. it's rare to see bright green and many players (myself included) have chat set to fade very quickly (or outright invisible) so normally when i see a scramble vote it's because of this. will consider though

Reply:
That’s fair, maybe having

```suggestion
	MC_PrintToChatAllEx(client, "{teamcolor}%s {default}wants to {green}scramble the teams{default}. [{lightgreen}%d{default}/{lightgreen}%d {default}votes required]", name, g_iVotes, g_iVotesNeeded);
```

where it emphasises “scramble” would make more sense then?

Reply:
The more colours and chat spam we have, the less "casual" feeling the game will feel. I already stripped *ALL* colours from all the plugins in this branch, except maybe some updated plugins i havn't got to yet maybe. 

If someone has chat time low/invis like that i expect them to understand how community servers work. Some people will just never look and there is nothing you can do to change that.

Reply:
`PrintCenterTextAll(“VOTE TO SCRAMBLE THE TEAMS NOW!”)`